### DISCUSSION TITLE
timers and interrupts

### DISCUSSION THREAD

**Discussion by dhruv221** [earlephilhower/arduino-pico#1605] [Ideas]
could timers, alarms and the corresponding interrupt features be added as they are required a lot for robots?

**Comment by earlephilhower** [comment]
You have full access to the Pico SDK for timer/alarms.  If there's an Arduino library you'd like to write/port, I'm sure the folks here can give pointers.  But I myself am unaware of anything close to a "standard library" for this kind of stuff, unfortunately.
